pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
+
+> Tool that allows making operations on multiple images ie. resizing, cropping, flipping, adding effects. Replaces the original file.
```

Comment:
I would stay with `ie.`. I also didn't know what `viz.` means, it may be unclear for some people. I think, `ie.` fits there well.